### ISSUE TITLE
[BUG] VoiceOver on Range should report a sensible status when it gains focus

### ISSUE THREAD

**Issue by chi-jyskebank-dk** [kirbydesign/designsystem#3775]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When the user navigates to the minimum-/maximum-labels of a Range slider, their values are read out loud without any context. 

## Describe how to reproduce the bug

1. Go to StackBlitz: https://stackblitz.com/edit/stackblitz-starters-wxb9gweu?file=src%2Fmain.ts
2. With VoiceOver enabled, navigate to the Range slider and minimum/maximum-labels
3. Notice that the minimum/maximum-labels are read out loud without giving the user any context

## Which Kirby version was used?
10.2.1

## What was the expected behavior?
When a Range slider gains focus, it should read out the current value of the slider as well as the minimum- and maximum-values so that their context is clear to the user. 

This could either be as a "single component" eg. like "Slider, 100000, minimum value 0, maximum value 500000", or as separate values "Slider, 100000", "Slider, minimum value 0", "Slider, maximum value 500000". The key point is, that is must be obvious to the user what the values 0 and 500000 refers to. 

## Add any screenshots
See attached screen recording

## Please complete the following information:
- OS: macOS 15.2 with VoiceOver
- Browser: Google Chrome

https://github.com/user-attachments/assets/3480f78a-946d-4193-ae2f-87fe2318aec4

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
